pact-tests/pact-tests/coin-v5.repl

```diff
@@ -126,12 +126,12 @@
 ; w/o capability
 (expect-failure
   "direct call to credit fails"
-  "cap not in scope"
+  "require-capability: not granted: "
```

Comment:
Note, we had t correct our output to match legacy pact, as they are used inside the our test suite (marmalade v2) which we include via submodules.